src/renderer/components/Settings/index.js

```diff
@@ -0,0 +1,139 @@
+// @ts-check
+import * as React from 'react'
+import { SettingsMenu } from './SettingsMenu'
+import { defineMessages } from 'react-intl'
+import MapIcon from '@material-ui/icons/MapOutlined'
+import InfoIcon from '@material-ui/icons/InfoOutlined'
+import { BackgroundMaps } from './BackgroundMaps'
+import { makeStyles } from '@material-ui/core'
+import Fade from '@material-ui/core/Fade'
+import Paper from '@material-ui/core/Paper'
+import { AboutMapeo } from './AboutMapeo'
+
+const m = defineMessages({
+  backgroundMap: 'Background Map',
+  aboutMapeo: 'About Mapeo'
+})
+
+const FADE_DURATION = 700
+
+/** @typedef {'BackgroundMap' | 'AboutMapeo'} SettingTabId */
+
+/** @type {import('./SettingsMenu').SettingsTabs[]} */
+const tabs = /** @typedef {const} */ [
+  {
+    tabId: 'BackgroundMap',
+    icon: <MapIcon />,
+    label: m.backgroundMap
+  },
+  {
+    tabId: 'AboutMapeo',
+    icon: <InfoIcon />,
+    label: m.aboutMapeo
+  }
+]
+
+/**
+ *
+ * @typedef SettingsProp
+ * @prop {boolean} reset
+ * @prop {function(boolean):void} setReset
+ * @prop {boolean} fadeIn
+ */
+
+/** @param {SettingsProp} props */
+export const Settings = ({ reset, setReset, fadeIn }) => {
+  const [menuVisible, setMenuVisibility] = React.useState(true)
+
+  /** @type {import('./SettingsMenu').SettingsTabs['tabId'] | false} */
+  const initialState = /** {const} */ (false)
+
+  const [tabValue, setTabValue] = React.useState(initialState)
+
+  const classes = useStyles()
+
+  if (reset) {
+    setReset(false)
+    if (tabValue === 'BackgroundMap') setTabValue(false)
+  }
```

Comment:
this state resetting code isn't necessary because you remount this component whenever it's navigated to i.e. in `Home.js` you do `tabIndex === 4 && <Settings ... />`

you don't need the `const [settingsReset, setSettingsReset] = React.useState(false)` in `Home.js` as a result and can remove the associated props from this component in that case.

tangentially, wonder if it makes sense to not conditionally render the Settings tab (like we do for the other app tabs) because that way you preserve the state when returning to it, which may be desirable from a UX perspective (as opposed to always starting in the reset state when navigating to it). This behavior could be useful when importing a map and you want to navigate elsewhere in the app without losing anything about the map import you started.

Reply:
So the reason for `setReset()` was if the user wanted to click "settings" again to reset. When background maps tab is open, it takes over the settings page completely. The `reset` variable made it possible for the user to click the settings tab again to go exit the background maps tab. That being said, if we don't want to conditionally render it, then we can't have both, since clicking "settings" will bring you back to where you were in the settings instead of resetting the state

Reply:
took out set reset and made the rendering NOT conditional: https://github.com/digidem/mapeo-desktop/pull/678/commits/73472a2872649cdd5aeff1f3772395226c25a63d

Reply:
ah i see. makes more sense to me now

> if the user wanted to click "settings" again to reset.

is this necessary given that the background maps settings screen has a back button in the sidebar? just thinking that adding the additional logic to cover this use case may not be worth it for now if a mechanism already exists

Reply:
it was slightly confusing to hit settings and see the background maps tab. But if we don't want to conditionally render it, then we have to get rid of `setReset()` (Or add even more complicated logic) anyways

Reply:
just remembered that you can reset the react state by providing a new `key` prop value, so it may be possible to avoid adding extra code and just providing a new value for the Settings tab panel `key` prop in `Home`.

Reply:
think it makes sense to avoid any reset behavior for now since it's less code to work with. if the reset stuff is desired, should be easy to add later on

Reply:
just to clarify, we can't really have both the "saving of the tab state", and "resetting settings on click". Either we click `settings` and it goes to the saved tab state Or we click `settings` and it resets the tab state

Reply:
yeah that's how I understood it and I'm leaning towards the former (preserve state) unless you're okay with adding the code to account for the latter